### DISCUSSION TITLE
First Contribution

### DISCUSSION THREAD

**Discussion by mariachrisochoou** [cgeo/cgeo#15469] [Q&A]
Hey, I really like this project and I was wondering whether there is a good first issue I can contribute to. Any tips for the first-comer? Thank you in advance!

**Comment by SammysHP** [comment]
A while ago we maintained a list of [simple tasks for new developers](https://github.com/cgeo/cgeo/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+first+issue%22). Unfortunately that list is pretty outdated by now. You could start [browsing the issue](https://github.com/cgeo/cgeo/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) and look for a topic that you are interested in. We can then check if that appears feasible for a beginner task.

Are you doing this as part of a university assignment? Would be good to know about your timeframe.

**Reply by mariachrisochoou** [reply]
Thank you for your quick answer! Yes, it is part of a university assignment, I need to be able to present the design of my contribution till the end of April and the finished result at the end of May.

**Comment by moving-bits** [comment]
@mariachrisochoou 
Are you still interested? There's a small issue which would be a task suitable for someone new to c:geo development, but would lead you to
- setting up an Android development environment
- using Git / GitHub
- dive a bit into the project structure of c:geo, searching the source for the relevant place to fix that issue
- add a string in an environment using CrowdIn-based translations
- creating a pull request (PR)
- ...

(surely with some pointers from the dev team)

see #15539

If you don't want to dive into that specific issue, or if you are no longer interested into doing your assignment based on c:geo, that's ok as well - just drop us a note then, please, so that we can reassign the mentioned issue.
